flask_unchained/bundles/cors/config.py

```diff
@@ -0,0 +1,17 @@
+from flask_unchained import BundleConfig
+
+
+class Config(BundleConfig):
+    # Default values; override as necessary
+    CORS_ORIGINS = '*'
+    CORS_METHODS = ALL_METHODS
```

Comment:
Upstream's default is already all methods so I think we can leave this out (the `ALL_METHODS` constant from `flask_unchained` is perhaps confusingly named, it doesn't include `HEAD`)